### ISSUE TITLE
Compilation failed: missing...

### ISSUE THREAD

**Issue by soehest** [kevinlekiller/Newznab-Blacklist#14]
Just testing out the new blacklists which looks awesome. But on update_binaries.php it will echo this error:
PHP Warning:  preg_match(): Compilation failed: missing ) at offset 271 in /var/www/localhost/htdocs/nnplus/www/lib/binaries.php on line 619
PHP Warning:  preg_match(): Compilation failed: lookbehind assertion is not fixed length at offset 27 in /var/www/localhost/htdocs/nnplus/www/lib/binaries.php on line 619

Further debugging (inserting echo $blist['ID']."debug\n"; at line 617) will show that problem lies in id's:
100001
100044

best regards

**Comment by soehest** [comment]
latest commit: https://github.com/kevinlekiller/Newznab-Blacklist/commit/c2fb5440baa71c5e01f954e7c237ae78fbad505e fixed id 100001 which no longer gives an error :-)

**Comment by thezoggy** [comment]
yeah, sadly php cant do negative lookbehinds with non fixed length.. which poses a problem on excluding some one off scenarios. im trying to get the nn guys to change how whitelists works so that we can just use that to counter some blacklist as currently a whitelist has no power against anything thats blacklisted.
